documentation/book/appendix_crds.adoc

```diff
@@ -1070,6 +1072,52 @@ Configuration of how TLS certificates are used within the cluster. This applies
 |string (one of [replace-key, renew-certificate])
 |====
 
+[id='type-KafkaExporterSpec-{context}']
+### `KafkaExporterSpec` schema reference
+
+Used in: xref:type-KafkaSpec-{context}[`KafkaSpec`]
+
+
+[options="header"]
+|====
+|Property                    |Description
+|image                1.2+<.<|The docker image for the pods.
+|string
+|groupRegex           1.2+<.<|Regex that determines which consumer groups to collect.
+|string
+|topicRegex           1.2+<.<|Regex that determines which topics to collect.
+|string
+|resources            1.2+<.<|Resource constraints (limits and requests).
+|xref:type-ResourceRequirements-{context}[`ResourceRequirements`]
+|logging              1.2+<.<|Only log messages with the given severity or above. Valid levels: [debug, info, warn, error, fatal].
+|string
+|enableSaramaLogging  1.2+<.<|Turn on Sarama logging.
```

Comment:
Would we ever want this disabled?

Reply:
Yes, because it might create a lot of undesired noise.